### PR TITLE
 Open version dependencies 

### PR DESCRIPTION
**Pull request by scoopex**
This morning i ran in various dependency problems because i have some modules which request release 7.0.1 of puppetlabs-apt.
This change opens the dependency a bit.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     },
     {
       "name": "puppetlabs/apt",
-      "version_requirement": ">= 2.2.0 < 7.0.0"
+      "version_requirement": ">= 2.2.0 < 7.1.0"
```
Comment (ekohl) [inline metadata.json]: Usually we follow semver and allow the entire major version
```suggestion
      "version_requirement": ">= 2.2.0 < 8.0.0"
```

Comment (scoopex) [reply metadata.json]: done

**Comment by ekohl** [comment]
Thanks! I've squashed it with a slightly more descriptive message.

**Comment by scoopex** [comment]
Can you publish a release for that? Then i can delete my fork :-)

**Pull request merged**